tests/PHPStan/Analyser/data/bug-6866.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Bug6866;
+
+use function PHPStan\Testing\assertType;
+
+function f(): void
+{
+	$str = str_repeat('a', 99);
+	assertType('"' . $str . '"', $str);
```

Comment:
Ah I mixed it up. The analyserIntegaration data/* files do not require the assertType calls. 

You put the code in there which should work as is - see other data/ files beeing analysed by the this TestCase

Reply:
ah ok. So, should I move the `assertType(...)` part of the test back to data/literal-string.php and leave only the standalone str_repeat() in here?

Reply:
Yes, I think so